tests_tf/test_attacks.py

```diff
@@ -777,5 +778,82 @@ def test_multiple_initial_random_step(self):
         self.assertTrue(np.mean(orig_labs == new_labs_multi) < 0.1)
 
 
+class TestFastFeatureAdversaries(CleverHansTest):
+    def setUp(self):
+        super(TestFastFeatureAdversaries, self).setUp()
+        import tensorflow as tf
+
+        def make_imagenet_cnn(input_shape=(None, 224, 224, 3)):
```

Comment:
How long does this test take to run? The model looks pretty big for a Travis test

Reply:
19 seconds isn't so bad. Is there anything trivial that can speed it up? For example, it looks like the batch size is 100. Does the test still work with batch size 1?

Does the attack still succeed on unrealistically small models, e.g. with only 32 filters per layer?

Reply:
Changed 100 to 10 in 58a7054 and it's faster now. Thanks for the suggestion.